expo/expo/urls.py

```diff
@@ -3,21 +3,36 @@
 from django.conf.urls.static import static
 from cms.sitemaps import CMSSitemap
 
+from expo.local_settings import APP_DJANGOBB
```

Comment:
should import this from settings (it might not be in local_settings, which causes an exception)